scripts/release/utils/git.js

```diff
@@ -30,23 +30,40 @@ exports.cleanTree = async () => {
   });
 };
 
+/**
+ * Commits the current state of the repository and tags it with the appropriate
+ * version changes.
+ *
+ * Returns the tagged commits
+ */
 exports.commitAndTag = async (updatedVersions, releaseType) => {
-  await git.add('*/package.json');
-  await git.commit(
-    releaseType === 'Staging' ? 'Publish Prerelease' : 'Publish'
+  await exec('git add */package.json');
```

Comment:
Revise the input args to this function. It doesn't look like `releaseType` is used any more.

Reply:
Done!!